tests/examples/example11-net-symbol.test.tsx

```diff
@@ -90,7 +90,7 @@ test("schematic net symbol", () => {
       <trace schDisplayLabel="GND2" from=".U1 > .pin13" to=".R1 > .pin2" />
       <trace schDisplayLabel="GND2" from=".U1 > .pin20" to=".R1 > .pin2" />
       <trace schDisplayLabel="GND2" from=".U1 > .pin28" to=".R1 > .pin2" />
-      <trace from=".U1 > .pin4" to=".R1 > .pin2" />
+      <trace schDisplayLabel="GND2" from=".U1 > .pin4" to=".R1 > .pin2" />
```

Comment:
lets revert this, now that i have the context it doesnt make sense to change the test this way

```suggestion
      <trace from=".U1 > .pin4" to=".R1 > .pin2" />
```

Reply:
@seveibar you said it's a redundant trace, should we make it automatically create a ground symbol if it's in a net with another ground symbol?

Reply:
maybe someone does want to draw the trace alongside the symbol,
and there is this case where two traces go into the same symbol, but this case only looks good if the ports are close to each other
![image](https://github.com/user-attachments/assets/c3189d90-410b-4b57-9ba4-c6383bd2b7fc)

Reply:
what do you think?